api_mt/Trados.md

```diff
@@ -0,0 +1,121 @@
+## 2. How to set up Trados Studio 2021™ plugins
+### 2.1 Amazon Translate (AWS)
+#### 2.1.1 How to obtain an API key
+Refer to [Point 4.1](#41-amazon-translate-aws) for details.
```

Comment:
These links are broken now, because it's on a different page.

Also, I don't think we should use "point" or numbered sections.  We can use numbers when there are steps that need to be done in order.

> ## How to set up machine translation plugins in Trados Studio 2021
>
> ### Amazon Translate (AWS)
>
> #### 1.API key
> [Get an Amazon Translate API key](/products/amazon-translate/api-key)
> 
> #### 2. Plugin installation
> 1. Open Trados Studio 2021
> 2. Go to **Add-ins > RWS AppStore**
> ...
> #### 2. Project settings
> 1. Go to **Project Settings**
> ...

Reply:
:-) During my studies, we had a long discussion as to whether such numbering should be used. If this is something you require then I shall delete the numbering, of course. But just to make sure: you would rather there were no: 2.0

Reply:
I think that the higher level numbers are unnecessary, since each CAT has its particular page.
Then 2.1, 2.1.1, 2.2 can be replaced by headers and subheaders (H1 and H3), and instructions can be numbered.

@bittlingmayer do you think this is a good approach?

Reply:
Yeah, I agree with Cecilia.  In this format, it just makes the deep links (anchor fragments) noisy, and is hard to maintain.